core/src/staked_nodes_updater_service.rs

```diff
@@ -36,11 +36,15 @@ impl StakedNodesUpdaterService {
                     let mut new_ip_to_stake = HashMap::new();
                     let mut new_id_to_stake = HashMap::new();
                     let mut total_stake = 0;
+                    let mut max_stake: u64 = 0;
+                    let mut min_stake: u64 = u64::MAX;
```

Comment:
This is computing `{min,max}_stake` but never actually updating those fields in `shared_staked_nodes`.
This is patched in https://github.com/solana-labs/solana/pull/31082

Until #31082 is merged, these two fields are always zero, so any benchmarks or tests done so far have been wrong.

Reply:
Sorry, I do not understand, the min max is updated in try_refresh_stake_maps line 76-82. What am I missing?

Reply:
The respective fields in `shared_staked_nodes` are not updated:
https://github.com/solana-labs/solana/blob/4564bcdc1/core/src/staked_nodes_updater_service.rs#L54-L57

Reply:
These fields:
https://github.com/solana-labs/solana/blob/03abaf76d/streamer/src/streamer.rs#L31-L32
are always zero.

Reply:
Got it. Thanks! The results of attached PDF of comparing different receive_window size was done via hard coded value in the code when I set the receive_window per connection -- as we did not have a mechanism to simulate the stakes in the bench-tps tool.